components/slices/banner.vue

```diff
@@ -0,0 +1,254 @@
+<template>
+  <section :class="`${bannerWithBackgroundImageClass}`">
+    <div :class="`banner ${bannerWithImageClass}`">
```

Comment:
On peut aussi faire comme ceci et s'affranchir de mettre la logique dans le js
```suggestion
    <div :class="['banner', {'banner-with-image': this.hasImage}]">
```